components/output/bp5758d.rst

```diff
@@ -0,0 +1,138 @@
+BP5758D LED driver
+==================
+
+.. seo::
+    :description: Instructions for setting up BP5758D LED drivers in ESPHome.
+    :keywords: BP5758D, DoHome G25 Globe bulb
+
+.. _bp5758d-component:
+
+Component/Hub
+-------------
+
+The BP5758D component represents a BP5758D LED diver chain in
+ESPHome. Communication is done with two GPIO pins (DATA and CLK).
+
+To use the channels of this components, you first need to setup the
+global ``bp5758d`` hub and give it an id, and then define the
+:ref:`individual output channels <bp5758d-output>`.
+It is used in some smart light bulbs:
+
+- DoHome G25 Globe bulb
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    bp5758d:
+      data_pin: GPIO3
+      clock_pin: GPIO5
+
+    # Individual outputs
+    output:
+      - platform: bp5758d
+        id: output_red
+        channel: 2 # Valid values 1-5 to match OUT1-OUT5
+        current: 10 # In mA, valid values 0-90, default 10
+      - platform: bp5758d
+        id: output_green
+        channel: 3 # Valid values 1-5 to match OUT1-OUT5
+        current: 10 # In mA, valid values 0-90, default 10
+      - platform: bp5758d
+        id: output_blue
+        channel: 1 # Valid values 1-5 to match OUT1-OUT5
+        current: 10 # In mA, valid values 0-90, default 10
+      - platform: bp5758d
+        id: output_coldwhite
+        channel: 5 # Valid values 1-5 to match OUT1-OUT5
+        current: 10 # In mA, valid values 0-90, default 10
+      - platform: bp5758d
+        id: output_warmwhite
+        channel: 4 # Valid values 1-5 to match OUT1-OUT5
+        current: 10 # In mA, valid values 0-90, default 10
```

Comment:
This example should only have the hub code, then below is a single float output. 
If you want to have a full example for a specific bulb, it should go down the bottom above `see also`
```suggestion
```